static/admin/config.yml

```diff
@@ -215,6 +215,293 @@ collections:
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                ],
+                        },
+                    ],
+            }
+          - {
+                label: 'Protocols',
+                name: 'protocols',
+                widget: 'object',
+                required: false,
+                collapsed: true,
+                fields:
+                    [
+                        {
+                            label: 'Diagrams',
+                            name: 'diagrams',
+                            widget: 'list',
+                            required: false,
+                            collapsed: true,
+                            fields:
+                                [
+                                    {
+                                        label: 'Title',
+                                        name: 'title',
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                    {
+                                        label: 'Image',
+                                        name: 'image',
+                                        widget: 'image',
+                                    },
+                                    {
+                                        label: 'Caption',
+                                        name: 'caption',
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                ],
+                        },
+                    ],
+            }
+          - {
+                label: 'Supplementary Information',
+                name: 'supplementary_info',
+                widget: 'list',
+                required: false,
+                fields:
+                    [
+                        {
+                            label: 'Acknowledgements',
+                            name: 'acknowledgements',
+                            widget: 'string',
+                            required: false,
+                        },
+                        {
+                            label: 'Supplementary Data',
+                            name: 'supplementary_data',
+                            widget: 'string',
+                            required: false,
+                        },
+                    ],
+            }
+          - {
+                label: 'STR Analysis',
+                name: 'str_analysis',
+                widget: 'object',
+                required: false,
+                collapsed: true,
+                fields: [
+                        {
+                            label: 'STR Analysis Results', #  todo: fix this, not sure exactly how to format this one yet
```

Comment:
I think the current structure works well for collecting data for the table, but I've checked several cell lines, data in `Donor fibroblasts` and `WTC line*` is identical. 

Ruge TODO: 1. What's a more specific name for label 2. Are the `Donor fibroblasts` and `WTC line*` columns always the same 3. Is the text in Amelogenin the same cross all WTC lines?